src/main/java/ro/siit/proiectgrupa17/controller/GuestController.java

```diff
@@ -62,4 +65,25 @@ public void deleteGuest(@PathVariable("id") int id) {
         guestService.deleteGuestById(id);
     }
 
+    @GetMapping("/deleteConfirmation/{id}")
+    public String deleteConfirmation(@PathVariable("id") int id, Model model) {
+        Guest guest = guestService.findById(id);
+        model.addAttribute("guest", guest);
+        return "deleteConfirmation";
+    }
+
+    @PostMapping("/guests/{id}/delete")
```

Comment:
lasa variabila ultima de obicei. Foloseste @DeleteMapping cand faci delete. Nu mai ai nevoie de /delete in path cand verbul http e delete